src/test/kotlin/no/nav/syfo/dialogmote/api/v2/AvlysMoteMedTidspunktTilbakeITidSpek.kt

```diff
@@ -0,0 +1,173 @@
+package no.nav.syfo.dialogmote.api.v2
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.ktor.http.*
+import io.ktor.http.HttpHeaders.Authorization
+import io.ktor.server.testing.*
+import io.mockk.*
+import no.nav.syfo.application.mq.MQSenderInterface
+import no.nav.syfo.dialogmote.api.domain.DialogmoteDTO
+import no.nav.syfo.dialogmote.database.getMoteStatusEndretNotPublished
+import no.nav.syfo.dialogmote.domain.DialogmoteStatus
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.testhelper.UserConstants.ARBEIDSTAKER_FNR
+import no.nav.syfo.testhelper.UserConstants.VEILEDER_IDENT
+import no.nav.syfo.testhelper.generator.*
+import no.nav.syfo.testhelper.mock.oppfolgingstilfellePersonDTO
+import no.nav.syfo.util.NAV_PERSONIDENT_HEADER
+import no.nav.syfo.util.bearerHeader
+import no.nav.syfo.brev.arbeidstaker.brukernotifikasjon.BrukernotifikasjonProducer
+import no.nav.syfo.dialogmote.domain.MotedeltakerVarselType
+import org.amshove.kluent.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDateTime
+
+class AvlysMoteMedTidspunktTilbakeITidSpek : Spek({
```

Comment:
Passer denne test inn i en test-klasse som allerede eksisterer?
Testene begynner å ta veldig lang tid å kjøre oppe(8-9 min med build) og for hver test vi har som kjører opp kjører ned hele testmiljøet så går det tregere enn om testen ligger i en fil som allerede gjør det.

Reply:
Skal sjekke om det er enkelt å slå den sammen med den andre test-klassen for avlyst møte.